event-timestamp/demo.js

```diff
@@ -0,0 +1,27 @@
+var previousX = 0;
+var previousY = 0;
+var previousT = 0;
+
+// While a 'mousemove' listener is being used for the purposes of this
+// example, the new timeStamp resolution applies to all Event types.
+window.addEventListener('mousemove', function(event) {
+  var Δx = event.screenX - previousX;
```

Comment:
:) keep it if you like it

On Thu, Jan 21, 2016, 7:37 AM Jeffrey Posnick <notifications@github.com>
wrote:

> In event-timestamp/demo.js
> <https://github.com/GoogleChrome/samples/pull/279#discussion_r50416143>:
>
> > @@ -0,0 +1,27 @@
> > +var previousX = 0;
> > +var previousY = 0;
> > +var previousT = 0;
> > +
> > +// While a 'mousemove' listener is being used for the purposes of this
> > +// example, the new timeStamp resolution applies to all Event types.
> > +window.addEventListener('mousemove', function(event) {
> > +  var Δx = event.screenX - previousX;
>
> Why are you conspiring to deprive me of the little joy Unicode has to
> offer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/samples/pull/279/files#r50416143>.
>